docs/source/data_view.rst

```diff
@@ -278,11 +398,19 @@ the |has_editor_value| method returns ``False``.
 
 .. |AbstractIndexManager| replace:: :py:class:`~pyface.data_view.index_manager.AbstractIndexManager`
 .. |AbstractDataModel| replace:: :py:class:`~pyface.data_view.abstract_data_model.AbstractDataModel`
-.. |DataViewSetError| replace:: :py:class:`~pyface.data_view.abstract_data_model.DataViewSetError`
+.. |AbstractDataExporter| replace:: :py:class:`~pyface.data_view.abstract_data_exporter.AbstractDataExporter`
+.. |AbstractValueType| replace:: :py:class:`~pyface.data_view.abstract_value_type.AbstractValueType`
+.. |DataFormat| replace:: :py:class:`~pyface.data_view.i_data_wrapper.DataFormat`
+.. |DataViewGetError| replace:: :py:class:`~pyface.data_view.data_view_errors.DataViewGetError`
+.. |DataViewSetError| replace:: :py:class:`~pyface.data_view.data_view_errors.DataViewSetError`
 .. |EditableValue| replace:: :py:class:`~pyface.data_view.value_types.editable_value.EditableValue`
+.. |FileDropHandler| replace:: :py:class:`~pyface.drop_handler.FileDropHandler`
 .. |IDataViewWidget| replace:: :py:class:`~pyface.data_view.i_data_view_widget.IDataViewWidget`
+.. |IDropHandler| replace:: :py:class:`~pyface.i_drop_handler.IDropHandler`
 .. |IntIndexManager| replace:: :py:class:`~pyface.data_view.index_manager.IntIndexManager`
 .. |IntValue| replace:: :py:class:`~pyface.data_view.value_types.numeric_value.IntValue`
+.. |ItemDataExporter| replace:: :py:class:`~pyface.data_view.exporters.item_data_exporter.ItemDataExporter`
+.. |RowDataExporter| replace:: :py:class:`~pyface.data_view.exporters.row_data_exporter.RowDataExporter`
```

Comment:
I think the modules and the classes have been renamed:
Doing a find-and-replace-all `item_data_exporter` -> `item_exporter`, `row_data_exporter` -> `row_exporter` etc seem to fix the missing links.

Reply:
> Doing a find-and-replace-all item_data_exporter -> item_exporter, row_data_exporter -> row_exporter etc seem to fix the missing links.

Or `data_exporter` -> `exporter`, and `DataExporter` -> `Exporter` :)